user_validation.py

```diff
@@ -0,0 +1,19 @@
+def usuario_validacion(usuario):
+    return usuario == "admin"
+
+def contrasena_validacion(contrasena):
+    return contrasena == "123456"
+

```

Comment:
Podemos extrar el admin y contraseña en constantes para que quede claro. Algo asi como: 

```suggestion
    MOCK_USERNAME = "admin"
    MOCK_PASSWORD = "123456"
    
    def usuario_validacion(usuario):
    return usuario == MOCK_USERNAME

  def contrasena_validacion(contrasena):
      return contrasena == MOCK_PASSWORD

```